src/common/components/card/CollectionCard.tsx

```diff
@@ -9,7 +9,7 @@ function CollectionCard({ id, title, infoLines, keywords, href, image }: Item) {
     <Card id={id}>
       <Card.Content>
         <Card.Title variant="h3" title={title} href={href} />
-        <Card.InfoLines infoLines={infoLines} />
+        <Card.InfoLines as="p" variant="body-l" infoLines={infoLines} />
```

Comment:
Variant `body-l` should be mapped to the `p` element by default:

https://github.com/City-of-Helsinki/liikunta-helsinki/blob/main/src/common/components/text/Text.tsx#L36